### DISCUSSION TITLE
ScriptReference/EditorWindow.Show

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#63] [Notes]
# ScriptReference/EditorWindow.Show

https://docs.unity3d.com/ScriptReference/EditorWindow.Show.html

**Comment by vertxxyz** [comment]
`Show` calls [`EditorGUIUtility.ExitGUI`](https://docs.unity3d.com/ScriptReference/GUIUtility.ExitGUI.html) which fires an exception which is caught by the GUI context. This will cause code running after this call to not run.
If you want to you can catch the [`ExitGUIException`](https://docs.unity3d.com/ScriptReference/ExitGUIException.html) and re-throw it once you're done.
